packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Basic.Example.tsx

```diff
@@ -68,4 +64,17 @@ export class ContextualMenuBasicExample extends React.Component {
       </div>
     );
   }
+
 }
+
+function _createItem(name: string): IContextualMenuItem {
+  const item: IContextualMenuItem = {
+    key: name,
+    name,
+    onClick: () => {
```

Comment:
This wouldn't show the break, to show the break it would need to return a value, so something like 
```
onClick: ()=> {
    console.log(`${name} clicked`);
    return 'Oh no I returned something';
}
```
would show the break